src/client/ui/SnackBar.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import styles from "./SnackBar.scss";
+import errorIcon from "../assets/error-icon.svg";
+import warningIcon from "../assets/warning-icon.svg";
+import PropTypes from "prop-types";
+import Button from "./Button";
+import classNames from "classnames";
+
+export default function SnackBar({ conflictType }) {
+  const conflictTypes = {
+    missing: {
+      img: errorIcon,
+      content: "Missing nodes in your .scene file.",
```

Comment:
Should this be "Missing nodes in inherited GLTF"?

Reply:
oh yeah, my bad.